src/components/Button.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Button = props => {
+  const { name } = props;
+  return (
+    <>
+      <button type="button">
+        {name}
+      </button>
+    </>
```

Comment:
- [ ] React Fragment it's not needed here since we are returning only one parent JSX element: `button`